### ISSUE TITLE
Impossibility to upload a file (attach a file) on GITLAB with UT phone in a bug report

### ISSUE THREAD

**Issue by gbdomubpkm** [ubports/morph-browser#206]
Impossibility  to upload a file (attach a file) on GITLAB with UT phone in a bug report.
I speak of screenshots.

Seen with  OTA 8 and 9.

It is possible that it is a specific bug already reported (or not) in a more technical way but I still prefer to underline it :

With NEXUS 5 totally impossible for me :  I can choose an image (screenshot)  once in the gallery, but it doesn't upload. Clicking the link a second time underlines this, but nothing happens,  it remains inactive.
![screenshot20190628_084516600](https://user-images.githubusercontent.com/50362604/60351404-06502200-99c6-11e9-9e1f-56eef5ce360a.png)
![screenshot20190628_084535084](https://user-images.githubusercontent.com/50362604/60351973-2b916000-99c7-11e9-8601-4a8d6f7086ac.png)

With the BQ E5 HD : I was able to upload a screenshot but I couldn't upload a second image just after (the link attach image remains inactive).

Not tested with other UT phones.

This could be annoying enough if there is no way to attach images (screenshots) in a reliable way for bug reports using its UT phone when  (IF)  Ubports will have completely switched to gitlab.

Attaching files with gitlab worked very well for me under vivid.

No problem to attach files in github with UT phones.

Thank you.

**Comment by balcy** [comment]
I can confirm, that on BQ 4.5 I can only upload one item only:
- click "Attach a file" 
- "Choose from" dialog appears
- select file from File manager or Gallery
- the dialog closes, the file is uploaded
- click "Attach a file"  
- "Choose from" dialog appears
- select file from File manager or Gallery
- the dialog closes, the file is not uploaded
- click "Attach a file"
- no dialog opens (onFileDialogRequested is not called)

Even reloading the page does not help, only restarting morph or opening the issue in another tab.
(Sometimes it is possible to upload more files)

**Comment by gbdomubpkm** [comment]
Hello.
I hope you can prioritize this problem because it is more and more annoying, gitlab being more and more used...
All my support. Thank you very much.

**Comment by gbdomubpkm** [comment]
I didn't have the opportunity to try but it is very likely that this bug is now fixed by the qt update.

**Comment by gbdomubpkm** [comment]
OPO dev after last update: I tried tonight and finally the problem is still there! 😒  A picture is uploaded,  then the upload link is frozen on the second picture.

**Comment by balcy** [comment]
yes I remembered trying it early this year with QtWE 5.13 or 5.14 without success :(

**Comment by gbdomubpkm** [comment]
I tried (only through webber webapps) with the Volla OTA-16 under github and gitlab.
Under Github, I could do the 'attach' operation only once (taking into account one or several files).
Under Gitlab, impossible from the first try.

I should have the same result with morph, yes ?
Quite annoying.

Edit : I am wrong. No problem with the Volla thru Morph-browser with Github and Gitlab. The problem of 'attaching' is with webber webapps.
